packages/components/src/List/ItemTitle/ItemTitle.component.js

```diff
@@ -146,7 +146,7 @@ function ItemTitle({ id, className, item, titleProps }) {
 	return (
 		<div className={theme['item-title']}>
 			{icon}
-			<TooltipTrigger label={value} tooltipPlacement="top">{titleElement}</TooltipTrigger>
+			<TooltipTrigger label={`${value}`} tooltipPlacement="top">{titleElement}</TooltipTrigger>
```

Comment:
add test showing that even if `value` is undefined, label will be an empty string